src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -84,61 +85,21 @@ protected void RenderRootComponent(int componentId)
         /// </summary>
         /// <param name="componentId">The ID returned by <see cref="AssignRootComponentId(IComponent)"/>.</param>
         /// <param name="initialParameters">The <see cref="ParameterCollection"/>with the initial parameters to use for rendering.</param>
-        protected void RenderRootComponent(int componentId, ParameterCollection initialParameters)
+        protected async void RenderRootComponent(int componentId, ParameterCollection initialParameters)
```

Comment:
Does this method really serve any purpose now? Seems like we could drop it and just make RenderRootComponentAsync be the only one.

Reply:
@SteveSandersonMS I'm fine with it, so would the recommendation for people be to simply not await the returned task if they don't want to wait for the entire render?

Reply:
@javiercn I guess so. There's no extra advantage in providing an `async void` wrapper around an `async Task` method, as far as I can tell.